client/stylesheets/_toCat.scss

```diff
@@ -2375,26 +2384,33 @@ i:hover {
 
 .headerAuth {
     margin-right: 20px !important;
+    margin-top: 13px !important;
 }
 
 .headerAuth li {
     margin-right: 5px;
 }
 
+.headerNotDrop {
+	margin: 5px !important;
+}
+
 .headerNotDrop a {
     color: $color-spacegrey !important;
+    padding: 12px !important;
 }
 
 .headerNotDrop a:hover {
     padding-bottom: 10px !important;
     color: white !important;
+    background-color: #404040 !important;
```

Comment:
Color literals like `#404040` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces